tests/ci/ci.py

```diff
@@ -1613,6 +1613,40 @@ def _upload_build_profile_data(
             logging.error("Failed to insert binary_size_file for the build, continue")
 
 
+def _add_build_to_version_history(
+    pr_info: PRInfo,
+    job_report: JobReport,
+    git_ref: str,
+    version: str,
+    docker_tag: str,
+    ch_helper: ClickHouseHelper,
+) -> None:
+    # with some probability we will not silently break this logic
+    assert pr_info.sha and pr_info.commit_html_url and version and git_ref
+
+    data = {
+        "check_start_time": job_report.start_time,
+        "pull_request_number": pr_info.number,
+        "pull_request_url": pr_info.pr_html_url,
+        "commit_sha": pr_info.sha,
+        "commit_url": pr_info.commit_html_url,
+        "version": version,
+        "docker_tag": docker_tag,
+        "git_ref": git_ref,
+    }
+
+    json_str = json.dumps(data)
+
+    print(f"::notice ::Log Adding record to versions history: {json_str}")
+
+    try:
+        ch_helper.insert_json_into(
+            db="default", table="version_history", json_str=json_str
+        )
+    except InsertException:
```

Comment:
It will always be silent with default `safe=True`, so it's unnecessary try/except.